test/lib/ufe/testComboCmd.py

```diff
@@ -706,8 +706,15 @@ def testFallback(self):
         mayaSphereItem        = ufe.Hierarchy.createItem(mayaSpherePath)
         usdSphereItem         = ufe.Hierarchy.createItem(usdSpherePath)
         usdFallbackSphereItem = ufe.Hierarchy.createItem(usdFallbackSpherePath)
-        usdSphere3d         = ufe.Transform3d.transform3d(usdSphereItem)
-        usdFallbackSphere3d = ufe.Transform3d.transform3d(usdFallbackSphereItem)
+        # For scene items with fallback transform ops, the transform3d()
```

Comment:
Fix latent bug exposed with new transform3d() implementation for Maya fallback stacks.  It is inappropriate to use transform3d() for reading and writing fallback transform ops; editTransform3d() must be used.